sdk/core/azure-core-perf/src/main/java/com/azure/core/perf/UploadFromFileTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.perf;
+
+import com.azure.core.perf.core.ServiceTest;
+import com.azure.perf.test.core.PerfStressOptions;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceAsyncClient;
+import com.azure.storage.blob.BlobServiceClient;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+public class UploadFromFileTest extends ServiceTest<PerfStressOptions> {
+    public UploadFromFileTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public Mono<Void> globalSetupAsync() {
+        return super.globalSetupAsync();
+    }
+
+    @Override
+    public Mono<Void> globalCleanupAsync() {
+        return super.globalCleanupAsync();
+    }
+
+    @Override
+    public void run() {
+        String blobName = "perfblobtest-" + UUID.randomUUID();
+
+        BlobServiceClient storageClient = blobServiceClientBuilder.buildClient();
+
+        BlobContainerClient blobContainerClient = storageClient.createBlobContainer("perfupload" + UUID.randomUUID());
+        BlobClient blobClient = blobContainerClient.getBlobClient(blobName);
+        blobClient.uploadFromFile(filePath, true);
```

Comment:
For both tests, and the sync and async variants, we'll want to either add a configuration flag to share the singleton HttpClient or to use a new one on each call. Or does it always share the HttpClient given which branch this PR is targeting?

Reply:
yes, Used different dependency versions, a locally compiled one vs already deployed. 

>we'll want to either add a configuration flag to share the singleton HttpClient 
Idk, if we support using a configuration flag to specify to use or not use a shared http client.